### DISCUSSION TITLE
The future of the IBAX

### DISCUSSION THREAD

**Discussion by Gargenchua** [IBAX-io/go-ibax#1862] [Ideas]
@ibaxio It is truly an amazing feeling to wait patiently and be among the leading supporters and investors of such a unique project. The project will continue to evolve over the next few years and the developer pioneers will enjoy this moment. I think the ecosystem with these robust and secure codes is really very reliable. We are looking forward to future of the IBAX.

**Comment by yolandadadada** [comment]
thanks for your support.
